2nd Blog for girls code Mk

gtl90 opened the pull request.

Commit:
2nd blog for girls code MK

Commit:
few changes and added a few lines

Commit:
changes to format and so img appears

gtl90 commented:
img path wrong
 white space added to headings to format correctly
added bullet list at top

Commit:
door poster jpg added

gtl90 commented:
Just a little more glitter - going to miss RGSoC

v-thakkar commented on blog/_posts/2018-09-17-Code-2-a-good-summer.md:
Hi, 'Anna' is there for 2 times. Did you mean 'Ana Sofia'? And did you also mean 'Vaishali', as I don't any RGSoC organizer named Valisha. :)

v-thakkar commented on blog/_posts/2018-09-17-Code-2-a-good-summer.md:
The image seems to be broken for some reason. Please fix that.

v-thakkar commented on blog/_posts/2018-09-17-Code-2-a-good-summer.md:
```diff
@@ -0,0 +1,138 @@
+title: Team GirlsCode MK - Code 2 a good summer
+layout: post
+created_at: Sat Sep 15 2018
+permalink: blog/2018-07-25-Code-2-a-good-summer
```
Hi @gtl90 ! Thanks for your PR. Lovely blog post. Please check the bunch of minor comments I have. 

Filename and permalink needs to be the same.

gtl90 committed:
Update 2018-09-17-Code-2-a-good-summer.md

Changed name of organisers of RGSoC, checked path of image, changed permalink reference

gtl90 committed:
Update 2018-09-17-Code-2-a-good-summer.md

changed permalink date to reflect filename

gtl90 commented:
I have updated the pull request not sure if this is correct way of doing or whether I should create a new pull request.  Let me know.  Thank you Vaishali :)

gtl90 committed:
changes made after feedback from pull request

gtl90 committed:
Merge branch 'gh-pages' of https://github.com/gtl90/summer-of-code into gh-pages

v-thakkar commented:
That's the correct way @gtl90 ! Looks good to me now. Thanks!

Commit:
Make changes to the date of the post and images
